jerry-core/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ message(STATUS "MEM_HEAP_SIZE_KB          " ${MEM_HEAP_SIZE_KB})
 set(INCLUDE_CORE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/api"
-    "${CMAKE_CURRENT_SOURCE_DIR}/debugger"
+    "${CMAKE_CURRENT_SOURCE_DIR}/dbg"
```

Comment:
We already have api, ecma, (j)mem, jrt, lit, js, re, vm. Actually, even jerry is an abbreviation. So, based on established precedent, I see nothing bad in abbreviations. Moreover, with the line length rules we have in place, "jerry_debugger" as a prefix takes more than 10% of the allowed spaced (compare it to "dbg"). Finally, it is very subjective, but DBG is quite a clear abbreviation for DeBuG or DeBuGging to me.